content-services/content-sources-backend
# Fixes 4552: fix missing method bug with mock gen

## Summary
This PR addresses the missing method bug (i.e. automates commenting out the `registry_mock.go` file) when generating mocks and updates the way we use the `mockery` tool to a new config as the old way be deprecated.

## Testing steps
- Verify that after adding a new method to a `dao` interface and to its "impl struct", `make mock` runs successfully without errors.
- Also `make mock` should now run without warnings about using an outdated config.
